### PR TITLE
Update README.md - Requires Python >= 3.11

### PR DESCRIPTION
**Pull request by adarshxs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ identification in GPT-2 small](https://arxiv.org/abs/2211.00593):
 
 ## Setup
 
-Follow these steps to install the repo.  You'll first need python/pip, as well as node/npm.
+Follow these steps to install the repo.  You'll first need python/pip, as well as node/npm. Requires `Python >= 3.11`
```
Comment (WuTheFWasThat) [inline README.md:34]: maybe merge with the second sentence?

Comment (adarshxs) [reply README.md:34]: yeah sure

Comment (stevenbills) [reply README.md:34]: The Python version requirement is already specified in `setup.py`. Is there a strong reason to specify it here as well?

Comment (adarshxs) [reply README.md:34]: Yeah its just a heads up in the readme. I ran into error of requiring python >= 3.11 for the neuron viewer after I activated my env and ran `pip install -e .`

Comment (adarshxs) [reply README.md:34]: Shall I close this @stevenbills ?

Comment (stevenbills) [reply README.md:34]: Yes please.

**Pull request closed without merging**